test/cpp_extensions/complex_registration_extension.cpp

```diff
@@ -80,6 +81,9 @@ TypeID ComplexCPUType::ID() const {
   return TypeID::ComplexCPU;
 }
 
+static auto complex_empty_registration = globalATenDispatch()
+  .registerOp(Backend::ComplexCPU, "aten::empty(int[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor", &ComplexCPUType::empty);
```

Comment:
If complex is really going to use `registerOp`, that makes me a lot more nervous about lack of error checking. (Maybe this should have been rewritten to use your other extension API?)